### PR TITLE
Fixes #202: Redundancy in uploading images

### LINKED ISSUE #202: Redundancy in uploading images
**Actual Behaviour**

The images are added again if selected from the image selector in the upload section.

**Expected Behaviour**

The uploaded image should not be readded. A unique hash should exist for each image.

**Steps to reproduce it**

1. Go to upload tab and upload image from the image selector.
2. After uploading, click on upload image button again.
3. The image selector opens, and same images can be added again.

**Screenshots of the issue**

Images 2 and 4 are same.

screenshot_2017-03-20-02-32-38-040_vn mbm phimp me

**Would you like to work on the issue?**

Yes, I would like to work on it.

### PR DESCRIPTION
**Pull request by rohanagarwal94**
Fixes issue #202 

Changes: 
No duplicate images will be uploaded now.

### CODE REVIEW

**Comment by phimpme-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">:tada: Thanks for the PR @rohanagarwal94 we hope you have changed the variables in the upload_apk_forks.sh its for easy moderation.</td>
    </tr>
  </tbody>
</table>

:link: Maintainers: the apk to test can be found at https://github.com/rohanagarwal94/phimpme-android/blob/apk/app-debug.apk

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: app/src/main/java/vn/mbm/phimp/me/Upload.java [hunk 1]
```diff
@@ -306,6 +311,15 @@ public void onClick(View v) {
                             public void onClick(DialogInterface dialog, int which) {
                                 if (!removableList.isEmpty()) {
                                     uploadGridList.removeAll(removableList);
+                                    for(String imagePath : removableList)
+                                    {
```
Comment (Abhi2424shek) [inline app/src/main/java/vn/mbm/phimp/me/Upload.java]: Pls put the bracket in a single line.

File: app/src/main/java/vn/mbm/phimp/me/utils/FileUtils.java [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+import android.content.Context;
+import android.text.format.Formatter;
+
+/**
+ * Created by rohanagarwal94 on 29/3/17.
+ */
+
+public class FileUtils {
+    private FileUtils() {
+
```
Comment (Abhi2424shek) [inline app/src/main/java/vn/mbm/phimp/me/utils/FileUtils.java]: Empty function body.

Comment (rohanagarwal94) [inline app/src/main/java/vn/mbm/phimp/me/utils/FileUtils.java]: @Abhi2424shek it is just the constructor of the class. Just trying to follow good coding practices.

**Comment by rohanagarwal94** [comment]
@pa1pal @Abhi2424shek please review. Made the necessary changes.

File: app/src/main/java/vn/mbm/phimp/me/utils/FileUtils.java (hunk 2 above)
Comment (Abhi2424shek) [inline app/src/main/java/vn/mbm/phimp/me/utils/FileUtils.java]: Constructor should be used to initialize variables and should not be left empty. @rohanagarwal94

**Comment by rohanagarwal94** [comment]
@pa1pal @Abhi2424shek please review. Made the necessary changes.

**Comment by mariobehling** [comment]
Good job! Thanks.

**Pull request merged**